code/modules/projectiles/ammo_datums.dm

```diff
@@ -2692,6 +2692,34 @@
 	shell_speed = AMMO_SPEED_TIER_2
 	added_spit_delay = 0
 
+
+/datum/ammo/xeno/acid/marking
+	name = "marking spit"
+
+	damage= 15
+	shell_speed = AMMO_SPEED_TIER_2
+	accuracy = HIT_ACCURACY_TIER_10*8
+	flags_ammo_behavior = AMMO_SKIPS_ALIENS
+	max_range = 32
+
+/datum/ammo/xeno/acid/marking/on_hit_mob(mob/M, obj/item/projectile/P)
+	. = ..()
+	M.AddComponent(/datum/component/bonus_damage_stack, 100, world.time)
```

Comment:
These take a very long time to decay. Spitters will be able to spam these and give everyone 10% damage vulnerability that's long-lasting. I'd recommend just making a new component that works similarly but clears it instantly after, say, 5 seconds.

Reply:
I think I figured out something that works; making the damage increase fade after like 1 second

Reply:
/datum/ammo/xeno/acid/marking/on_hit_mob(mob/Target, obj/item/projectile/P)
	. = ..()
	Target.AddComponent(/datum/component/bonus_damage_stack, 100, 1 SECONDS)
	for(var/mob/living/carbon/Xenomorph/X in range(5, Target))